srcpkgs/swirc/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'swirc'
+pkgname=swirc
+version=3.2.3
+revision=1
+build_style=configure
+make_install_args="PREFIX=/usr"
+hostmakedepends="pkg-config which"
+makedepends="libcurl-devel libressl-devel ncurses-devel libidn-devel"
+checkdepends="cmocka-devel"
+short_desc="Curses ICB and IRC client"
+maintainer="Markus Uhlin <markus.uhlin@bredband.net>"
+license="BSD-3-Clause, ISC, MIT"
+homepage="https://www.nifty-networks.net/swirc"
+distfiles="https://www.nifty-networks.net/swirc/releases/${pkgname}-${version}.tgz"
+checksum="52e273bbc9ea6384841947858d60cb740a65bd80d2257c7cab39d9a4eb919f06"
+
+post_configure() {
+	local _file="options.mk"
+
+	if [ ! -f "$_file" ]; then
+		echo "post_configure: error: cannot find $_file"
```

Comment:
You can use `msg_error("error: cannot find $_file\n")` here and omit the `return 1`
This prints the message in red and exits `xbps-src`.

You can omit the stage name because it is printed in the line just before the error.

Reply:
Thanks! Will fix that.

Reply:
Now fixed